docs/content/operations/insert-segment-to-db.md

```diff
@@ -94,3 +94,27 @@ and `druid-hdfs-storage` in the extension list.
 
 After running this command, the segments table in `mysql` should store the new location for each segment we just inserted.
 Note that for segments stored in HDFS, druid config must contain core-site.xml as described in [Druid Docs](http://druid.io/docs/latest/tutorials/cluster.html), as this new location is stored with relative path.
+
+It is also possible to use `s3` as deep storage. In order to work with it, specify `s3` as deep storage type and load 
+[`druid-s3-extensions`](../development/extensions-core/s3.html) as an extension.
+
+```
+java
+-Ddruid.metadata.storage.type=mysql 
+-Ddruid.metadata.storage.connector.connectURI=jdbc\:mysql\://localhost\:3306/druid 
+-Ddruid.metadata.storage.connector.user=druid 
+-Ddruid.metadata.storage.connector.password=diurd
```

Comment:
It was copied from the hdfs section in the same document. I guess that it was intended because of a password.